datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -158,6 +161,8 @@ internal class QuestionnaireViewModel(application: Application, state: SavedStat
         }
       }
     }
+    // Add extension for questionnaire launch time stamp
+    questionnaireResponse.extension.add(Extension("http://github.com/google-android/launch-timestamp", DateTimeType(Date())))
```

Comment:
```suggestion
    questionnaireResponse.extension.add(Extension("http://github.com/google-android/questionnaire-launch-timestamp", DateTimeType(Date())))
```
To specify the extension namespace.